docs/content/docs/scale.md

```diff
@@ -0,0 +1,110 @@
+# Scale and limits
```

Comment:
Not sure if it's really required, but all the other docs for the website have front matter at the very top for Hugo, like https://github.com/openservicemesh/osm/blob/09af915f64605dc331bd924d085ac7403032e7f0/docs/content/docs/development_guide.md#L1-L5

Reply:
I will add, I just saw on another doc.